main/views.py

```diff
@@ -1,118 +1,13 @@
 from django.shortcuts import render, redirect, reverse
-from django.core.mail import send_mail
-from main.models import UserProfile, Stock, Transaction, NewsPost
+from main.models import UserProfile, Stock, Transaction, StockPurchased, NewsPost
 from django.http import HttpResponse, HttpResponseRedirect
-from django.contrib.auth import authenticate, login, logout
```

Comment:
removed the unnecessary imports for authentication that we don't require in views.py now.